db/articles.js

```diff
@@ -0,0 +1,103 @@
+class Articles {
+  constructor() {
+    // Creates the Article array
+    this._articleList = [
+      {
+        title : '50 Shades Of Ray',
+        body : "From #000000 to #FFFFFF and more",
+        author : 'E.L. James',
+        urlTitle : '50%20Shades%20Of%20Ray'
+      },
+
+      {
+        title : 'The Girl With The Baseem Tattoo',
+        body : '1 Fish, 2 Fish. Red Fish, Dish gurl',
+        author : 'Stieg Larsson',
+        urlTitle : 'The%20Girl%20With%20The%20Baseem%20Tattoo'
+      },
+
+      {
+        title : 'Go The Fuck To Sleep',
+        body : "The cats nestle close to their kittens now. The lambs have laid down with the sheep. You’re cozy and warm in your bed, my dear",
+        author : 'Adam Mansbach',
+        urlTitle : 'Go%20The%20Fuck%20To%20Sleep'
+      }
+    ];
+  }
+
+  // Lists all objects in the array
+  showAll() {
+      return this._articleList;
+  }
+
+  // First checks if the title already exists, and if it does, it does not create
+  createArticle(info) {
+    if (this.checkIfTitleExists(info.title)) {
+      return false;
+    } else {
+      let addArticle = {
+        title : info.title,
+        body : info.body,
+        author : info.author,
+        urlTitle : encodeURI(info.title)};
+      this._articleList.push(addArticle);
+      return true;
+    }
+  }
+
+  // Checks if the element exists in the array
+  checkIfTitleExists(title) {
```

Comment:
Change function name to `_checkIfTitleExists`.  The underscore is a convention that denotes that it is a helper function to be used only inside of this module.